_locales/it/messages.json

```diff
@@ -0,0 +1,102 @@
+{
+  "name": {
+    "message": "Tabs aside",
+    "description": "name of the extension. Displayed in the manifest and pane header"
+  },
+  "description": {
+    "message": "Il classico menu di Microsoft Edge \"Tabs Aside\"",
+    "description": "Extension description"
+  },
+  "author": {
+    "message": "Michael \"XFox\" Gordeev",
+    "description": "Author name"
+  },
+  "options": {
+    "message": "Opzioni",
+    "description": "Alternative text for options button in the pane"
+  },
+  "closePanel": {
+    "message": "Chiudi",
+    "description": "Alternative text for close panel button"
+  },
+  "loadOnRestore": {
+    "message": "Riapri le schede al ripristino",
+    "description": "Label for option"
+  },
+  "showDeleteDialog": {
+    "message": "Mostra una finestra di conferma prima di eliminare qualcosa",
+    "description": "Label for option"
+  },
+  "swapIconAction": {
+    "message": "Sposta le finestre in Tabs Aside al click sull'estensione (%TOGGLE_SHORTCUT% o click con il tasto destro sull'icona dell'estensione per aprire il pannello)",
+    "description": "Label for option"
+  },
+  "github": {
+    "message": "Visita la pagina di GitHub",
+    "description": "Link title"
+  },
+  "contributors": {
+    "message": "Contribuenti al progetto",
+    "description": "Link title"
+  },
+  "feedback": {
+    "message": "Lascia un feedback",
+    "description": "Link title"
+  },
+  "buyMeACoffee": {
+    "message": "Comprami un caffè!",
+    "description": "Link title"
+  },
+  "credits": {
+    "message": "Sviluppato da Michael 'XFox' Gordeev",
+    "description": "Options menu credits"
+  },
+  "setAside": {
+    "message": "Sposta le schede correnti in Tabs Aside",
+    "description": "Save collection action name. Used in the pane, extension context menu and manifest shortcuts"
+  },
+  "nothingSaved": {
+    "message": "Non hai finestre salvate qua",
```

Comment:
A bit different translation. Google Translate claims that the word "finestre" has only one definition - "windows". That means this word in inappropriate for the situation since we save browser tabs, not windows

Reply:
I'll replace it with schede (tabs in Italian)